src/util/sendEmail.ts

```diff
@@ -0,0 +1,68 @@
+import transporter from '../config/email'
+import { User } from '../components/users/user'
+
+interface Email {
+  subject: string
+  body: string
+  link?: string
+  linkTitle?: string
+}
+
+export const sendEmail = (recipients: User[], email: Email) => {
+  if (process.env.NODE_ENV === 'production') {
+    recipients.forEach(user => {
+      transporter.sendMail({
+        from: `TanulóSCH <${process.env.EMAIL_USER}>`,
+        to: user.email,
+        subject: email.subject,
+        text: email.body,
+        html: `<html>
+                  <body style="min-height: 50vh;color: rgba(31, 41, 55, 1);
+                              background: rgb(191,219,254);
+                              background: linear-gradient(180deg, rgba(191,219,254,1)
+                               0%, rgba(221,214,254,1) 100%);
+                              font-family: Arial, Helvetica, sans-serif;">
+                      <div style="
+                                  margin-bottom: 2rem;margin-right: 1rem;">
+                          <h1 style="display: inline-block; width: auto; padding-left: 1.5rem;
+                                     padding-right: 1.5rem; padding-top:0.75rem;
+                                     padding-bottom: 0.75rem; margin-bottom: 0px;
+                                     font-size: 1.5rem; line-height: 2rem; font-weight: 700;
+                                      background-color: rgba(249, 250, 251, 1);
+                                      margin-left: -1rem;">TanulóSCH</h1>
+                      </div>
+                      <div style="text-align: center;
+                                padding-left: 2rem;
+                                padding-right: 2rem;padding-top: 1.5rem;padding-bottom: 2rem;
+                                margin-left: 1.5rem;margin-right: 1.5rem;border-radius: 1rem;
+                                background-color: rgba(249, 250, 251,1);
+                                --tw-shadow: 0px 4px 32px rgba(0, 0, 0, 0.3),
```

Comment:
I'm not really sure using tailwind CSS variables here. Also, the styles can be drastically reduced by size, e.g. combining the margin/padding definitions into their shorthand. Or maybe define the CSS in a separate string variable and inject it into the HTML like this (not sure if it works exactly like this):
```
const emailStyles = `
.some {
  css: 'here';
}`

const emailTemplate = `
<html>
<style>
${emailStyles}
</style>
// content here
</html>`
```

Reply:
I don't think email clients accept style tags, but I tried making the file look better in another way.
Also, I don't really understand how the box-shadow thing works, but most email clients don't allow it either, and it looks fine without it.